internal/test/wmcb/wmcb_test.go

```diff
@@ -112,10 +112,17 @@ func TestWMCB(t *testing.T) {
 	for _, vm := range framework.WinVMs {
 		log.Printf("Testing VM: %s", vm.GetCredentials().InstanceId())
 		wVM := &wmcbVM{vm}
-		files := strings.Split(*filesToBeTransferred, ",")
-		for _, file := range files {
-			err := wVM.CopyFile(file, remoteDir)
-			require.NoError(t, err, "error copying %s to the Windows VM", file)
+		dirs := strings.Split(*dirsToBeTransferred, ",")
+		// all the files in `/payload` directory will be transferred to `remoteDir`
+		// for files not in `/payload`, new directory will be created in `remoteDir` which resembles the linux folder path
+		// for eg. `/payload/cni` will be stored in `remoteDir\cni` and `/foo/bar` will be stored in `remoteDir\foo\bar`
+		for _, dir := range dirs {
+			// remove `/payload` from the path and split the path
+			dirPath := strings.Split(strings.Replace(dir, "/payload", "", -1), "/")
+			// join the path with `\\` and place it in `remoteDir`
+			remotePath := remoteDir + "\\" + strings.Join(dirPath, "\\")
+			err := wVM.CopyDirectory(dir, remotePath)
+			require.NoError(t, err, "error copying %s to the Windows VM", dir)
```

Comment:
I am a nack on this approach. You are assuming that the input will be `/payload`. Sure we are building the image that way but you cannot leak such assumptions into the test code.